src/Defer/Defer.php

```diff
@@ -40,12 +42,12 @@ class DeferExtension extends GraphQLExtension
     /**
      * @var bool
      */
-    protected $defer = true;
+    protected $deferMore = true;
```

Comment:
What does this actually do? 
It feels weird when reading it.

Reply:
Renamed to `$acceptFurtherDeferring`